js/bytetrade.js

```diff
@@ -0,0 +1,1000 @@
+        return this.parseTrades (response, market, since, limit);
+    }
+
+    parseOrder (order, market = undefined) {
+        const status = this.safeString (order, 'status');
+        const symbol = this.findSymbol (this.safeString (order, 'symbol'), market);
+        const timestamp = this.safeInteger (order, 'timestamp');
+        const datetime = this.safeString (order, 'datetime');
+        const lastTradeTimestamp = this.safeInteger (order, 'lastTradeTimestamp');
+        const price = this.safeFloat (order, 'price');
+        const amount = this.safeFloat (order, 'amount');
+        const filled = this.safeFloat (order, 'filled');
+        const remaining = this.safeFloat (order, 'remaining');
+        const cost = this.safeFloat (order, 'cost');
+        const average = this.safeFloat (order, 'average');
+        const id = this.safeString (order, 'id');
+        const type = this.safeString (order, 'type');
+        const side = this.safeString (order, 'side');
+        const fee = this.safeValue (order, 'fee');
+        return {
+            'info': order,
+            'id': id,
+            'timestamp': timestamp,
+            'datetime': datetime,
+            'lastTradeTimestamp': lastTradeTimestamp,
+            'symbol': symbol,
+            'type': type,
+            'side': side,
+            'price': price,
+            'amount': amount,
+            'cost': cost,
+            'average': average,
+            'filled': filled,
+            'remaining': remaining,
+            'status': status,
+            'fee': fee,
+        };
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        if (this.apiKey === undefined) {
+            throw new ArgumentsRequired ('fetchDeposits requires hasAlreadyAuthenticatedSuccessfully');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        // the next 5 lines are added to support for testing orders
+        let sideNum = undefined;
+        let typeNum = undefined;
+        if (side === 'sell') {
+            sideNum = 1;
+        } else {
+            sideNum = 2;
+        }
+        if (type === 'limit') {
+            typeNum = 1;
+        } else {
+            typeNum = 2;
+        }
+        const baseRealName = market['base'].split ('@')[0];
+        const quoteRealName = market['quote'].split ('@')[0];
+        const marketName = baseRealName + '/' + quoteRealName;
+        const baseId = market['baseId'];
+        const baseCurrency = this.currencies_by_id[baseId];
+        const amountFloat = parseFloat (amount);
+        const amountChain = amountFloat * Math.pow (10, baseCurrency['basePrecision']);
+        const quoteId = market['quoteId'];
+        const quoteCurrency = this.currencies_by_id[quoteId];
+        const priceFloat = parseFloat (price);
+        const priceChain = priceFloat * Math.pow (10, quoteCurrency['basePrecision']);
```

Comment:
please use `priceToPrecision` here

Reply:
Private API is not ready now.
The data sent to the DEX chain must be an integer, priceToPrecision seem not do this.
There are a lot of question in Private API, and have sent a email to info@ccxt.trade for help ：）